packages/frontend/src/components/LightdashVisualization/VisualizationProvider.tsx

```diff
@@ -236,10 +256,12 @@ const VisualizationProvider: FC<React.PropsWithChildren<Props>> = ({
              */
             return isGroupedSeries(seriesLike)
                 ? calculateSeriesColorAssignment(seriesLike)
-                : colorPalette[seriesIndex % colorPalette.length] ??
-                      getDefaultSeriesColor(seriesIndex);
+                : fallbackColors[
```

Comment:
This is basically what was done in different places already, this change centralizes it in `VisualizationProvider` and exposes `getSeriesColor` as the single source of truth (ish, there's still some more cleanup to be done)